common/SC_ServerBootDelayWarning.h

```diff
@@ -0,0 +1,2 @@
+void startServerBootDelayWarningTimer();
```

Comment:
add `#pragma once` to the top of this file, and move the comment from the top of the cpp file to this file instead. header files are typically the place where API documentation goes. you could also add a quick note about how these two calls are used if you want.

Reply:
thanks !